k8s/plugin/src/resources/objects.rs

```diff
@@ -0,0 +1,323 @@
+                .into_vec(),
+                ..Default::default()
+            },
+            PolicyRule {
+                api_groups: Some(vec![""].into_vec()),
+                resources: Some(vec!["nodes"].into_vec()),
+                verbs: vec!["get", "list"].into_vec(),
+                ..Default::default()
+            },
+            PolicyRule {
+                api_groups: Some(vec![""].into_vec()),
+                resources: Some(vec!["secrets"].into_vec()),
+                verbs: vec![
+                    "get",
+                    "list",
+                    "watch",
+                    "create",
+                    "delete",
+                    "deletecollection",
+                    "patch",
+                    "update",
+                ]
+                .into_vec(),
+                ..Default::default()
+            },
+            PolicyRule {
+                api_groups: Some(vec!["rbac.authorization.k8s.io"].into_vec()),
+                resources: Some(vec!["roles"].into_vec()),
+                verbs: vec!["create", "list", "delete", "get", "patch"].into_vec(),
+                ..Default::default()
+            },
+            PolicyRule {
+                api_groups: Some(vec!["rbac.authorization.k8s.io"].into_vec()),
+                resources: Some(vec!["rolebindings"].into_vec()),
+                verbs: vec!["create", "list", "delete", "get", "patch"].into_vec(),
+                ..Default::default()
+            },
+            PolicyRule {
+                api_groups: Some(vec!["rbac.authorization.k8s.io"].into_vec()),
+                resources: Some(vec!["clusterroles"].into_vec()),
+                verbs: vec!["create", "list", "delete", "get", "patch"].into_vec(),
+                ..Default::default()
+            },
+            PolicyRule {
+                api_groups: Some(vec!["rbac.authorization.k8s.io"].into_vec()),
+                resources: Some(vec!["clusterrolebindings"].into_vec()),
+                verbs: vec!["create", "list", "delete", "get", "patch"].into_vec(),
+                ..Default::default()
+            },
+            PolicyRule {
+                api_groups: Some(vec![""].into_vec()),
+                resources: Some(vec!["services"].into_vec()),
+                verbs: vec!["create", "list", "delete", "get", "patch"].into_vec(),
+                ..Default::default()
+            },
+            PolicyRule {
+                api_groups: Some(vec!["storage.k8s.io"].into_vec()),
+                resources: Some(vec!["storageclasses"].into_vec()),
+                verbs: vec!["create", "list", "delete", "get", "patch"].into_vec(),
+                ..Default::default()
+            },
+            PolicyRule {
+                api_groups: Some(vec!["storage.k8s.io"].into_vec()),
+                resources: Some(vec!["configmaps"].into_vec()),
+                verbs: vec!["create", "list", "delete", "get", "patch"].into_vec(),
+                ..Default::default()
```

Comment:
I think you have removed the wrong one. Configmaps are supposed to be a part of core api group as per the latest api-reference. cc @niladrih @avishnu @tiagolobocastro 

https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.25/#configmap-v1-core